validation-frameworks/functional/test_endpoint.py

```diff
@@ -7,19 +7,24 @@ def test_publish():
 
 @given("a running EKS cluster")
 def running_eks_cluster():
-    # Perform any necessary setup steps, such as deploying Prometheus in your EKS cluster
+    # Perform any necessary setup steps, such as deploying components in your EKS cluster
     pass
 
-#### WHEN RUNNING THIS PLEASE PORT FORWARD to 9090 ####
 @when("I check endpoints")
-def check_prometheus_endpoint():
+def check_eks_endpoint():
     eksclient = boto3.client('eks', region_name = 'us-east-1')
-    resp = eksclient.describe_cluster(name='blue')
+    resp = eksclient.describe_cluster(name='fleet-hub-cluster')
```

Comment:
Do not hardcode the cluster name make sure an environment variable `AWS_EKS_CLUSTER_NAME` is set before starting tests

Reply:
Updated the code to fetch the region from AWS_REGION and cluster name from EKS_CLUSTER_NAME environment variables.